mfuerstenau/gradle-buildconfig-plugin
# Upgrade the gradle version of build config plugin  to 7.3.2

# Feature: 
1. Upgrade the gradle version to 7.3.2 for mitigating for log4j vulnerability in Gradle builds
2. Support agp7.0

# How to test
- Comment several codes as follows:
![image](https://user-images.githubusercontent.com/16274345/146755295-54937d1c-f841-4b2d-be86-3ea645d6c8c7.png)
![image](https://user-images.githubusercontent.com/16274345/146755344-b67dee92-2ed2-45e5-b2f6-f48def2d3c8a.png)

-  execute `./gradlew publishToMavenLocal` task to generate local build-config plugin

- uncomment previous commented code, and execute `build` task, and you'll see generated code `test-build-config-agp7/build/gen/buildconfig/src/main/com/example/test/build/config/agp7/BuildConfig.java`  
![image](https://user-images.githubusercontent.com/16274345/146755874-f67ad120-5f5b-4df6-87c0-fb75e671a81f.png)
